app/gui/src/presenter/graph/state.rs

```diff
@@ -492,6 +492,7 @@ impl<'a> ControllerChange<'a> {
             None | Some(Value) => None,
             Some(DataflowError { trace }) => Some((Kind::Dataflow, None, trace)),
             Some(Panic { message, trace }) => Some((Kind::Panic, Some(message), trace)),
+            Some(Pending { message: _, progress: _ }) => Some((Kind::Panic, Some("Pending...".to_string()), Vec::new())),
```

Comment:
You can also write `Some(Pending { .. })` as a pattern, if you don't care about any field.

Reply:
OK. I'll keep that in mind. I leave the line unchanged, you will need to update this code to display something better in the IDE anyway.

Reply:
@JaroslavTulach do we want to highlight pending expressions as panics in IDE? I.e. we may want to return a separate object here (with a todo comment to implement handling of it later)